receiver/statsdreceiver/transport/client/client.go

```diff
@@ -7,33 +7,25 @@ import (
 	"fmt"
 	"io"
 	"net"
+	"strings"
 )
 
 // StatsD defines the properties of a StatsD connection.
 type StatsD struct {
-	Host string
-	Port int
-	Conn io.Writer
+	transport string
```

Comment:
why string and not `type Transport string` as defined in transport.go?

Reply:
Circular dependencies.